src/routes/index.tsx

```diff
@@ -4,20 +4,26 @@ import { RouteObject } from 'react-router-dom';
 import DashBoardScreen from 'screens/Dashboard';
 import SignInScreen from 'screens/SignIn';
 
-// TODO Update dashboard path to root and apply redirect if the user haven't logged in
+import ProtectedRoute from './ProtectedRoute';
+
 export const paths = {
   root: '/',
-  dashboard: '/dashboard',
+  signIn: '/sign-in',
 };
 
 const routes: RouteObject[] = [
   {
-    path: paths.root,
-    element: <SignInScreen />,
+    element: <ProtectedRoute />,
+    children: [
+      {
+        path: paths.root,
+        element: <DashBoardScreen />,
+      },
+    ],
   },
   {
-    path: paths.dashboard,
-    element: <DashBoardScreen />,
+    path: paths.signIn,
```

Comment:
Do we need the test here? Maybe the user is not logged in and the path is `paths.signIn`?

Reply:
@nvminhtue I'm having trouble finding ways to test this file. As a solution, I've written an integration test to confirm the behavior of "the user is not logged in and the path is paths.signIn" in [this PR](https://github.com/manh-t/react-survey/pull/62/files#diff-d299c7915109b3ddc4015b3d2427383bb98b029e9d81069c3fb2e2975a0c3baeR34). Do you think this is a sufficient way to test it, or do you have any other suggestions? :pray:

Reply:
I see, but the test in the mentioned PR doesn't cover the `paths.signIn` assertion like the way `paths.dashboard` did. But as long as you cover that case, it's okay to me, just placing tests for a single feature in different PRs is hard to follow though.

Reply:
@nvminhtue yeah, that's why I ask if you have any suggestions for testing this file 😄 Otherwise, I will try this case in the integration test. Anw, thank you 🙏